File: 0x03_Array/10807.cpp
#include <iostream>
#include <vector>

using namespace std;

int main()
{
    int N;
    int v;
    int arr[201] = {0};
    
    cin >> N;
    for (int i=0; i<N; i++)
    {
        int number = 0;
        cin >> number;
        arr[100 + number] += 1;
    }
    cin >> v;

    cout << arr[100 + v] << endl;

    return 0;
}

File: python/0x03_Array/10807.py
import sys
input = sys.stdin.readline

def solution():
    n = int(input().strip())
    arr = list(map(int, input().strip().split()))
    v = int(input().strip())

    count = 0
    for number in arr:
        if number == v:
            count += 1

    print(count)

if __name__ == "__main__":
    solution()